modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib

```diff
@@ -1,15 +1,3 @@
-@Article{hales15homogenization,
```

Comment:
@aeslaughter I've got a Catch-22 that I don't know how to resolve and would like to ask for your help: with this citation included both here and in the heat conduction module I get a documentation warning about duplicate bib entries, and without this citation entry I get a documentation failure. What a good path forward to resolve this (admittedly minor minor annoyance)? Should I put the entry in a combined bib file?

Reply:
There are two solutions:

1. You can move the reference into a framework level bib file, somewhere in `moose/framework/doc/content` and just avoid the need for the duplicate

2. In tensor_mechanics/doc/config.yml, you can add a configuration item as follows:

```yaml
MooseDocs.extensions.bibtex:
     duplicates:
        - hales15homogenization
```

Reply:
Also, FYI, for now warnings are allowed. Since we are adding so much for SQA the warnings are a mechanism to allow for changes to go in. Hopefully, in the near future, I will go back and fix up these little nuances as well, but they are not worth holding up adding content.